packages/plugins/opentelemetry/src/index.ts

```diff
@@ -131,6 +135,17 @@ export const useOpenTelemetry = (
       if (!operationAst) {
         return;
       }
+
+      const shouldReport = options.excludedOperationNames
+        ? typeof options.excludedOperationNames === 'function'
+          ? !options.excludedOperationNames(operationAst.name?.value)
+          : !options.excludedOperationNames.includes(operationAst.name?.value || '')
+        : true;
+
+      if (!shouldReport) {
+        return;
+      }
```

Comment:
Having tow ternary like this is a bit difficult to read. Since the result is just used to return, I do think it's a bit easier like this, what do you think ?

```suggestion
      if (options.excludedOperationNames) {
        if (typeof options.excludedOperationNames === 'function')
          if (!options.excludedOperationNames(operationAst.name?.value)) {
            return
          }
        } else if (!options.excludedOperationNames.includes(operationAst.name?.value || '')) {
          return
        }
      }
```

or even something like this, but a bit weird to have a ternary inside a if condition ^^'
```suggestion
      if (options.excludedOperationNames) {
        if (typeof options.excludedOperationNames === 'function'
          ? !options.excludedOperationNames(operationAst.name?.value)
          : !options.excludedOperationNames.includes(operationAst.name?.value || '')
        ) {
          return
        }
      }

Reply:
i picked the in-between solution :D